### PR TITLE
[All - end] Update to Compose 1.2.0

### PR DESCRIPTION
**Pull request by astamato**

### CODE REVIEW

File: AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt [hunk 1]
```diff
@@ -165,11 +146,11 @@ fun PostCardPopular(
     modifier: Modifier = Modifier
 ) {
     Card(
-        shape = MaterialTheme.shapes.medium,
-        modifier = modifier.size(280.dp, 240.dp),
         onClick = { navigateToArticle(post.id) },
+        shape = MaterialTheme.shapes.medium,
         // Step 2: Click labels
-        onClickLabel = stringResource(id = R.string.action_read_article)
+//        onClickLabel = stringResource(id = R.string.action_read_article),
```
Comment (astamato) [inline AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:152]: Could you help me figure out the correct change here? cc @JolandaVerhoef

Comment (JolandaVerhoef) [reply AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:152]: What's the failure?

Comment (astamato) [reply AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:152]: Won't compile with DeprecationLevel.ERROR. Suggests alternative constructors that don't have onClickLabel

File: AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt [hunk 2]
```diff
@@ -19,30 +19,11 @@ package com.example.jetnews.ui.home
...
-import androidx.compose.material.Card
-import androidx.compose.material.ContentAlpha
-import androidx.compose.material.ExperimentalMaterialApi
-import androidx.compose.material.Icon
-import androidx.compose.material.IconButton
-import androidx.compose.material.LocalContentAlpha
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
+import androidx.compose.foundation.layout.*
```
Comment (JolandaVerhoef) [inline AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:22]: I don't think you have the correct AS settings, as you're replacing all these imports with wildcard imports

File: AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt (hunk 1 above)
Comment (JolandaVerhoef) [reply AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:152]: Got it. The codelab will have to be adapted (code and text). I can do that!

File: AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt (hunk 2 above)
Comment (astamato) [reply AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt:22]: Ups :D wrong config, thanks!!

File: AdvancedStateAndSideEffectsCodelab/app/src/androidTest/java/androidx/compose/samples/crane/home/HomeTest.kt [hunk 3]
```diff
@@ -16,13 +16,10 @@
 
 package androidx.compose.samples.crane.home
 
-import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.*
```
Comment (astamato) [inline AdvancedStateAndSideEffectsCodelab/app/src/androidTest/java/androidx/compose/samples/crane/home/HomeTest.kt:19]: revert

**Pull request closed without merging**